Adjust documentation for upcoming 3.4 release (#130)
zammad/zammad-documentation: install/update.rst

Source update
=============

> 1. Ensure your System meets the requirenments
> ---------------------------------------------
> 
> Before updating your installation, please ensure that your system  meets Zammads current 
> :doc:`Software requirenments </prerequisites/software>`.

Reviewer: This seems like a strange thing to add to update instructions. Can dependencies change across a minor version bump? I would consider that a **breaking change**, which would require a major version bump according to [semantic versioning](https://semver.org/).

In any case, here's a rewording. If the above is true, consier adding this sentence just before "Before proceeding...":

> Dependencies may change with each update.

```suggestion
1. Check your dependencies
--------------------------

Before proceeding, double-check that your system environment
matches :doc:`Zammad’s requirements </prerequisites/software>`.
```

2. Download Zammad to your system
---------------------------------

> You can directly download Zammad from https://ftp.zammad.com/. If you want to ensure you have the latest stable version, 
> you can also get Zammad directly from `Github <https://github.com/zammad/zammad/archive/stable.zip>`_.

Reviewer: ```suggestion
Get the latest stable release of Zammad `here <https://github.com/zammad/zammad/archive/stable.zip>`_,
or find an older version at https://ftp.zammad.com.
```
